dvc/repo/get_url.py

```diff
@@ -16,6 +16,7 @@ def get_url(url, out=None):
 
     (dep,) = dependency.loads_from(None, [url])
     (out,) = output.loads_from(None, [out], use_cache=False)
-    dep.save()
+    if not dep.exists:
```

Comment:
Sorry, I totally forgot that the issue itself was about `exist` not working for some urls :slightly_frowning_face: So this won't actually solve it. My mistake, sorry for the confusion.

Ok, let me see if the exception approach mentioned above is feasible to implement...

Reply:
At least, this approach work for #4251 . I have tested.

Reply:
How is the current situation？

Reply:
Didn't get to research this yet. Need to confirm this and we'll merge if it is indeed fixed. Looks good as is. Thanks!

Reply:
@hl-a-k Thanks for your patience! Indeed, it was the `get_hash` that was causing that issue, so the current solution works like charm. Thank you! :pray: